File: 03.business-os/_archive/commonos_legacy_skeleton_20260425_184835/app_local/PocketSecretary/iphone/PocketSecretary/CommonOSBootstrap/CommonOSBootstrapRegistry.swift
import Foundation

enum CommonOSBootstrapRegistry {
    static func createDefault() -> CommonOSBootstrapDescriptor {
        CommonOSBootstrapDescriptor(
            appName: "PocketSecretary",
            launchMode: "commonos_bootstrap",
            uiOwner: "CommonOS",
            businessCanonOwner: "PocketSecretary",
            runtimeRoute: "runtime/commonos",
            entryRoute: "runtime/commonos/entry",
            contractRoute: "runtime/commonos/contract"
        )
    }
}

File: 03.business-os/PocketSecretary/android/app/src/main/java/com/lsam/PocketSecretary/runtime/commonos/bootstrap/CommonOSBootstrapRegistry.java
package com.lsam.PocketSecretary.runtime.commonos.bootstrap;

public final class CommonOSBootstrapRegistry {

    public CommonOSBootstrapDescriptor createDefault() {
        return new CommonOSBootstrapDescriptor(
                "PocketSecretary",
                "commonos_bootstrap",
                "CommonOS",
                "PocketSecretary",
                "runtime/commonos",
                "runtime/commonos/entry",
                "runtime/commonos/contract"
        );
    }
}
